js/events/core/events_engine.js

```diff
@@ -614,6 +614,18 @@ eventsEngine.hover = {
     }
 };
 
+eventsEngine.visibility = {
+    on: ($el, shown, hiding, { namespace }) => {
+        eventsEngine.on($el, addNamespace('dxhiding', namespace), hiding);
```

Comment:
Most of the quotes in this file are double. For consistency, it is better to use either double or single quotes in all files.